src/mvba/vcbc/mod.rs

```diff
@@ -1,16 +1,10 @@
 pub(super) mod context;
 pub(super) mod message;
-pub(super) mod state;
 
-mod deliver;
```

Comment:
I can see you deleted the state files. I prefer to not. Let's break the code and make logic separate per state.